qcfractal/storage_sockets/sqlalchemy_socket.py

```diff
@@ -2408,7 +2408,7 @@ def check_lib_versions(self):
             else:
                 current = db_ver.first()
 
-        return current.to_dict(exclude=['id'])
+            return current.to_dict(exclude=['id'])
```

Comment:
Feels a bit awkward to return inside a `with` statement. Can we case these to dict inside we `with` and then return outside?

Reply:
ok, I'll change and merge